### ISSUE TITLE
Windows Not Responding

### ISSUE THREAD

**Issue by Nigenteki** [Cockatrice/Cockatrice#4956]
**System Information:**

Windows 10 Home
_______________________________________________________________________________________

This issue seemingly only applies to when I am attempting to play a game, though I don't particularly use any other feature of Cockatrice so I can't be sure of that.

So starting a few days ago, Cockatrice has been periodically not responding; every 15 seconds or so, for about 5-7 seconds it would stop responding before fixing itself and allowing me to play. This was occurring before the latest patch of Cockatrice (2.9.0) but continued when I did. The issue seemingly improved slightly, but only allowing me slightly more time to play before not responding again. I checked with a tech savvy friend of mine, and it doesn't seem to be a connectivity or speed issue. They think it might be some interaction between my PC and the program, but they're unable to help from that point. I'm wondering if a recent Windows update may be interfering with the program, but that's only my best guess.

Thank you for your consideration.
_______________________________________________________________________________________

Unknown, just connecting seems to cause the bug.

**Comment by ebbit1q** [comment]
in what way does the application stop responding? do menus stop opening or does you moving a card take really long to happen?

**Comment by Nigenteki** [comment]
The program itself stops responding; I'm not sure why it triggers, it seems maybe triggered by something being tapped, or might just happen randomly, but the entire program stops responding, ctrl+alt+del style: i can't click on the program and it freezes, the window says "(not responding)"; after a few moments it will start responding again. Thankfully, it doesn't reset anything I do, seemingly, it just freezes, but it only seems to be affecting me right now. It's not having the same problem with my friend, with who I play frequently.

**Comment by ebbit1q** [comment]
do you have enough disk space available or another resource shortage? what server are you connecting to?

**Comment by Nigenteki** [comment]
I connect to Rooster Ranges, and I don't think so? I have 357 GB free in my C drive, and my memory is 44% on a given moment taken up mostly by Chrome and Steam (i don't always have Steam open) taking up about 500 MB of memory. I also checked my performance, and I don't have a resource shortage anywhere.

**Comment by ebbit1q** [comment]
without being able to reproduce your issue I don't know how we can figure out what is causing it, let alone find a fix. have you tried uninstalling and reinstalling the application from our release page?

**Comment by Nigenteki** [comment]
I originally had this problem in the old version of Cockatrice; I did attempt to fix this by uninstalling and reinstalling the latest version, and then I uninstalled and reinstalled, and in doing so I think i found the problem. With the first uninstall and reinstall process, I only uninstalled Cockatrice itself; card sets, settings, and other things were left alone. The second uninstall/install section, I wiped everything; and that fixed it because everything was default. The issue was fixed, everything was great! Until I activated sound again; as soon as I did that and tapped for mana, immediate return to the unresponsive lag situation. Turned the sound off and on to confirm, and I am very confident that the sound was the problem; for whatever reason, the program freezes for a few moments whenever a sound wants to be played. I would like the sound back, but I can play without it, though I would like to submit this as like, an issue in case more people might have this problem.

Regardless, thank you for your help! I really appreciate it. If you respond with a possible fix, I would appreciate it, but otherwise I would consider this resolved.

**Comment by ebbit1q** [comment]
ah, thank you for finding the cause of the issue, I still can't reproduce it myself though...

which one of the windows downloads did you use? (copy the text above the line in help>show debug log on a fresh start of cockatrice and it will tell me)

**Comment by Nigenteki** [comment]
Sorry about late response, busy holidays (and happy holidays to you, by the way).  The info:
Client Version: 2.9.0 (2023-09-14)
Client Operating System: Windows 10 Version 2009
Build Architecture: 64-bit
Qt Version: 5.15.2
System Locale: en_US
Install Mode: Standard

I couldn't tell you what i was using before that for sure, all I can say is that it was happening on whatever version I was using then as well.

**Comment by ebbit1q** [comment]
ah, I see you used the win7+ build, we also have a build for win10 and higher, could you try installing it from our release page to see if it fixes your issue?

**Comment by Nigenteki** [comment]
Yeah! I installed the Win 10 version and I've played a few games with it, and the glitch has disappeared entirely. Thank you for the help.
